content/en/notebooks/notebooks_new.md

```diff
@@ -0,0 +1,351 @@
+
+Click the icons on the cell action tray to adjust the size, alignment, add a caption for the image cell, or view the image in full screen mode.
+
+<!-- TODO Add updated image from new notebooks -->
+
+#### Markdown editor
+
+This approach places the image inline with your text, but does not provide options for resizing the image.
+
+Enter the edit mode on any Markdown cell and use any of the following options to add the image:
+- Drop an image file into the text cell area.
+- Copy and paste the image directly into the text cell area.
+- Hyperlink an external image using the image widget in the header or by referencing the [official Markdown guide][16]. **Note**: This option does not upload the image to be hosted by Datadog. 
+
+You can preview the image in the Preview tab before saving it to your notebook.
+
+## Adding comments to a notebook
+
+You can add comments on content in the body of the notebook. To comment on text, highlight the text and click on the comment icon in the toolbar. 
+
+<!-- TODO Add updated image from new notebooks -->
+
+To comment on a graph or an image, select the cell and click on the comment icon to the right of the cell.
+
+| Feature                  | Description                                                                                                          |
+|--------------------------|----------------------------------------------------------------------------------------------------------------------|
+| **Navigating to Comments** | Saved comments appear in the right margin of the notebook. Click a comment highlight in the text to align it, or click a comment in the margin to scroll to its location. |
+| **Responding to Comments** | Respond to comments by clicking on them in the right margin, which opens a comment box. You can write text, `@mention` a Datadog user, or resolve a comment by clicking **Resolve**. |
+| **Linking to Comments**    | Link to a specific comment by clicking the link icon in the top right corner of the comment to copy its link.      |
+| **Editing or Deleting Comments** | Edit or delete your comments by clicking the three-dot ellipsis menu in the top right corner of the comment.                 |
+| **Comment Notifications** | By default, email notifications are sent to the notebook author for new comments by others. Users in a comment thread receive notifications for each reply. To adjust notifications, in the cog menu, select **Notifications**. |
+
+## Multiplayer experience in Notebooks
+
+Notebooks are collaborative, allowing multiple users to edit simultaneously. If a collaborator opens your document, their cursor will appear inside the document. Hover over a cursor to see the collaborator's name.
+
+<!-- TODO Add updated image from new notebooks -->
+
+### Widgets
+
+When another user is editing a widget, an outline appears to indicate this. Since widgets are saved as last write wins, avoid editing a widget that someone else is working on.
```

Comment:
Small rephrase so this doesn't read as "the outline appears to be indicating this, but it might actually be indicating that"
```suggestion
When another user is editing a widget, an outline appears around the widget. Since widgets are saved as last write wins, avoid editing a widget that someone else is working on.
```